common/src/test/java/com/revenuecat/purchases/common/AnonymizerTest.kt

```diff
@@ -50,10 +50,29 @@ class AnonymizerTest {
 
     // endregion
 
+    // region anonymizeStringMap
+
+    @Test
+    fun `anonymizedStringMap anonymizes string fields if needed`() {
+        val originalMap = mapOf(
+            "key-1" to "string with some.pii@revenuecat.com and 192.168.1.1.",
+            "key-2" to "string without pii",
+            "key-3" to "string with other.pii@revenuecat.com"
+        )
+        val expectedMap = mapOf(
+            "key-1" to "string with ***** and *****.",
+            "key-2" to "string without pii",
+            "key-3" to "string with *****"
```

Comment:
thought for the future - it'd probably be useful to know what these were before getting anonymized, i.e.: having them be something like `redacted@***.com` or `redacted_ip`, etc